app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+  before_action :authenticate_user!
+
+  def index
+    rents = Rent.all
+    render json: rents
+  end
+
+  def create
+    rent = Rent.new(rent_params)
+    if rent.save
+      render json: rent
+    else
+      render json: rent.errors
```

Comment:
This will render with an ok status though

Reply:
Also, not a correction but a different way to write this would be something like
```
rent = Rent.new(rent_params)
render json: rent.errors unless rent.save # Use a different status though
render json: rent
```